opm/simulators/wells/VFPHelpers.hpp

```diff
@@ -258,6 +258,10 @@ inline InterpData findInterpData(const double& value_in, const std::vector<doubl
         }
     }
 
+    if (retval.factor_ > 3.0) {
+        retval.factor_ = 3.0;
+    }
+
```

Comment:
This limits the extrapolation I assume. Why 3 not 1? Maybe add a comment for the reasoning behind this particular choice.

Reply:
Indeed it limits extrapolation. I first used 1.0, but with 3.0 it performed better on the 1B case. I have not tested other values, and it could be tuned, I do not know if there is an optimal value.

Reply:
We can change this later. I suggest just add a comment for now.